games-arcade/gnake/files/gnake-0.94b-gcc15.patch

```diff
@@ -0,0 +1,39 @@
+https://bugs.gentoo.org/944252
+
+--- a/gnake.c
++++ b/gnake.c
+@@ -64,14 +64,14 @@ typedef struct {
+ void getOptions(int argc, char *argv[]);
+ void DrawWindow();
+ void Init();
+-void Step();
++void Step(int);
+ void Draw();
+ void End();
+ 
+ coord getRandEmptyCell(int mandatory);
+ void createApple(int i);
+ 
+-void Resize();
++void Resize(int);
+ 
+ char *TheGameName = "Gnake 0.94b";
+ 
+@@ -349,7 +349,7 @@ void CPUMove(int n) {
+   Snk[n].Dir = d;
+ }
+ 
+-void Step() {
++void Step(int) {
```

Comment:
Unnamed parameters are a C23ism but supported as an extension for older GCC at least, we should avoid that where possible for now especially for upstreamable patches and also as I'm not sure which versions of Clang permit it.

Reply:
Fixed